extensions/resteasy-classic/resteasy-client/runtime/pom.xml

```diff
@@ -58,6 +58,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-multipart-provider</artifactId>
+        </dependency>
```

Comment:
Not that I mind, but I would like to know why this is now necessary

Reply:
Also if it's necessary, we might have to depend on the `quarkus-resteasy-multipart` extension instead.

Reply:
This is the dependency from RESTEasy that provides the `EntityPart.Builder` implementation. Note that for the REST Client Classic, I've tried to keep it as the original implementation: https://github.com/resteasy/resteasy-microprofile/commit/bb8b433fc28d0be2560d21d1204eec07603b9bf7